en/install.md

```diff
@@ -9,7 +9,7 @@ table_of_contents: true
 
 To run the Snap Store Proxy, you will need:
 
-* A server running Ubuntu 20.04 LTS or newer on AMD64.
+* A server running one of the [currently supported LTS versions of Ubuntu](https://ubuntu.com/about/release-cycle) on AMD64.
```

Comment:
With this change it means now 16.04 and 18.04 are included in the list of releases that the snap should work?
Unless you mean the LTS standard support (3y) vs extended of 5y

Reply:
oops, completely misunderstood the page, standard is 5y and ESM is +5y